nix/devtools.nix

```diff
@@ -25,7 +25,7 @@ let
       }
       ''
         while true; do
-          (! ${silver-searcher}/bin/ag -l . | ${entr}/bin/entr -dr "$@")
+          (! ${silver-searcher}/bin/ag -l main src test | ${entr}/bin/entr -dr "$@")
```

Comment:
I needed to change this to support `postgrest-watch postgrest-coverage`, since the coverage script adds files on the top level directory. This causes an infinite loop of restarting through the watch script, even though those files are not monitored directly. This is because the `-d` option to entr watches changes to the top-level directory as soon as we watch any file from that directory.

The only file that is relevant to watch for changes would be `postgrest.cabal`. So right now, this is not the case and a restart with watch running has to be triggered manually when changing this file. Since we can only have one or the other, I thought supporting `postgrest-watch postgrest-coverage` makes more sense, because - at least in theory - it should be used more often ;).